💀 
```go
jreader/token_reader_default.go:22:7: const `isEasyJSON` is unused (unused)
const isEasyJSON = false //nolint:deadcode
```

@cwaldren-ld ok I'll open up a PR I guess with that other idea, moving these to test-only files.  I also had ran golangci-lint 1.50 locally rahter than what your CI is running (1.48), so I'll take a few minutes first to try to reproduce that failure here

--- a/jreader/token_reader_default.go
+++ b/jreader/token_reader_default.go
@@ -17,10 +17,6 @@ import (
 	"unicode/utf8"
 )
 
-// isEasyJSON is used in tests to e.g. expect different allocation behavior depending
-// on which backend is in use.
-const isEasyJSON = false //nolint:deadcode
-
 var (
 	tokenNull  = []byte("null")  //nolint:gochecknoglobals
 	tokenTrue  = []byte("true")  //nolint:gochecknoglobals

--- a/jreader/token_reader_default_test.go
+++ b/jreader/token_reader_default_test.go
@@ -1,0 +1,8 @@
+//go:build !launchdarkly_easyjson
+// +build !launchdarkly_easyjson
+
+package jreader
+
+// isEasyJSON is used in tests to e.g. expect different allocation behavior depending
+// on which backend is in use.
+const isEasyJSON = false

--- a/jreader/token_reader_easyjson.go
+++ b/jreader/token_reader_easyjson.go
@@ -16,10 +16,6 @@ import (
 	"github.com/mailru/easyjson/jlexer"
 )
 
-// isEasyJSON is used in tests to e.g. expect different allocation behavior depending
-// on which backend is in use.
-const isEasyJSON = true //nolint:deadcode
-
 type tokenReader struct {
 	// We might be initialized either with a pointer to an existing Lexer, in which case we'll use that.
 	pLexer *jlexer.Lexer

--- a/jreader/token_reader_easyjson_test.go
+++ b/jreader/token_reader_easyjson_test.go
@@ -1,0 +1,8 @@
+//go:build launchdarkly_easyjson
+// +build launchdarkly_easyjson
+
+package jreader
+
+// isEasyJSON is used in tests to e.g. expect different allocation behavior depending
+// on which backend is in use.
+const isEasyJSON = true

